2019azheng,2018fawan
  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . . . . . . . .
3 . . . . . . . .
4 . . . o @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
2-2

  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . . . . . . . .
3 . . . @ . . . .
4 . . . @ @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
4-1

  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . . . . . . . .
3 . . o @ . . . .
4 . . . o @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
3-3

  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . . . . . . . .
3 . @ @ @ . . . .
4 . . . o @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
5-2

  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . o . . . . . .
3 . @ o @ . . . .
4 . . . o @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
4-4

  1 2 3 4 5 6 7 8
1 . . . . . . . .
2 . o . . . . . .
3 . @ o @ . . . .
4 . . @ @ @ . . .
5 . . . @ o . . .
6 . . . . . . . .
7 . . . . . . . .
8 . . . . . . . .
6-3

winner:2019azheng won,'2018-02-27 19:21:02,774:DEBUG:[JAILED]:Listed student folders successfully\n2018-02-27 19:21:02,774:DEBUG:[JAILED]:Got data 2018fawan 5.0 o ???????????........??.o......??.@o@....??..@@@...??...@o...??........??........??........???????????\n2018-02-27 19:21:02,774:DEBUG:[JAILED]:Data is ok\n2018-02-27 19:21:02,795:INFO:[JAILED]:There is an error\nTraceback (most recent call last):\n  File "/home/othello/tourney/run_ai.py", line 64, in strat_wrapper\n    self.strat(board, player, best_shared, running)\n  File "/home/othello/tourney/students/2018fawan/strategy.py", line 195, in best_strategy\n    move = playMove(othBoard,othToken)\n  File "/home/othello/tourney/students/2018fawan/strategy.py", line 65, in playMove\n    startend=cache.index(1)+1\nValueError: 1 is not in list\n2018-02-27 19:21:02,796:DEBUG:[JAILED]:Got move -1\n'